modules/modal/react/lib/ModalContent.tsx

```diff
@@ -11,12 +11,12 @@ export interface ModalContentProps extends React.HTMLAttributes<HTMLDivElement>
    * The padding of the Modal. Accepts `zero`, `s`, or `l`.
    * @default PopupPadding.l
```

Comment:
that makes sense!

Reply:
hmm so when I remove them as optional, in the test it says it required those properties, kinda curious why because the parent adds them as default, thoughts?

Reply:
It's because `Modal` extends this interface. You'll have to defined them as optional in `ModalProps`